install/installing-with-kata-manager.md

```diff
@@ -43,5 +43,120 @@ Use the following command to only install Kata Containers packages.
 $ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/tests/master/cmd/kata-manager/kata-manager.sh) install-packages"
 ```
 
+## Validate the installation
```

Comment:
We try very hard to avoid duplication in the documentation. Since this command is already mentioned in the runtime's readme, please could you change this to something like the following (since the referenced documentation has some important further information about the `kata-check` command):

> To validate the installation, run the [`kata-check` command](https://github.com/kata-containers/runtime#hardware-requirements). To obtain a detailed list of configuration values, run the [`kata-env` command](https://github.com/kata-containers/runtime#configuration).

Also, although it can be useful, we try very hard to avoid displaying the *output* of commands, particularly those commands that produce a lot of output (see https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks). As such, I'd remove the `kata-env` output.

Finally, please can you add "Validate the installation" to the table of contents links at the top of this page? We have [a tool that can generate this for you](https://github.com/kata-containers/tests/tree/master/cmd/check-markdown#generate-a-toc), but there are many alternatives I believe.